README.md

```diff
@@ -1,16 +1,42 @@
-# Title
+# Meal Choice Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Define a method that takes in required and optional arguments
+- Use `puts` to output a string
+- Define a return value with the meal
 
 ## Introduction
 
-## SWBAT 1
+We've demonstrated the basics of defining methods, printing output, and
+returning values. Let's combine the concepts we've shown you to create a "Meal
+Choice" in our Ruby program.
 
-## SWBAT 2
+## Define a Method that Takes in Required and Optional Arguments
+
+We're attending a wedding and the newly weds-to-be want to know what types of food they should provide at the banquet. Define a method named `meal_choice` that has **three** parameters:
+
+- Two _required_ parameters for vegetables. Ex: `veg1` and `veg2`
+- And an _optional_ parameter for `protein` with a default value of `meat`
+
+## Use `puts` to output a string
+
+Whether you're omnivorous, vegetarian, or vegan, you're going to be eating a
+nutritious meal!
+
+In the body of your method, have the program `puts` "What a nutritious meal!"
```

Comment:
I was a bit confused b/c this doesn't use any of the parameters. We use them again in the second `puts`, but i think we can prime the reader's mind by setting up the expectation that there's a second thing to do.

Maybe say that "we're going to use puts twice". 


**First** let's (tell tell the user that they're getting a nutritious meal....)

**Second** let's make use of those paramters. Let's...